src/components/register/RegisterPage.js

```diff
@@ -9,33 +9,46 @@ import {
   Label,
   Map,
   Title,
-} from "./RegisterPageElements";
-import MapImg from "../../assets/images/maps2.jpeg";
-import Swal from "sweetalert2";
-import { postUsers } from "../../services/User.services";
+} from "./RegisterPageElements"
+import Swal from "sweetalert2"
+import { postUsers } from "../../services/User.services"
+import "mapbox-gl/dist/mapbox-gl.css"
+import mapboxgl from "mapbox-gl/dist/mapbox-gl.js"
+// eslint-disable-next-line import/no-webpack-loader-syntax
```

Comment:
Not sure what this is for

Reply:
Thanks Miguel. The css and js are respective of mapbox, they are lines that must be implemented according to the mapbox documentation. And the comment is so that it doesn't give error in the eslint.